internal/rows/errors.go

```diff
@@ -7,6 +7,8 @@ var errRowsNilRows = "databricks: nil Rows instance"
 var errRowsUnknowRowType = "databricks: unknown rows representation"
 var errRowsCloseFailed = "databricks: Rows instance Close operation failed"
 var errRowsMetadataFetchFailed = "databricks: Rows instance failed to retrieve result set metadata"
+var errRowsOnlyForward = "databricks: Rows instance can only iterate forward over rows"
```

Comment:
general question, should we give more info about the error, eg, what is the row number which has error?